packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -44,18 +47,20 @@ function estimateSize() {
 export const Combobox = ({ options, onChange, value, ...restProps }: ComboboxProps) => {
   const [items, setItems] = useState(options);
   const selectedItem = useMemo(() => options.find((option) => option.value === value) || null, [options, value]);
-  const listRef = useRef(null);
+  const inputRef = useRef<HTMLInputElement>(null);
+  const floatingRef = useRef(null);
 
-  const styles = useStyles2(getStyles);
+  const theme = useTheme2();
+  const styles = getComboboxStyles(theme);
```

Comment:
```suggestion
  const styles = useStyles2(getComboboxStyles);
```
`useStyles2` handles memoization etc.